architecture/core_concepts/projects_and_users.adoc

```diff
@@ -134,3 +134,12 @@ endif::[]
 Developers and administrators can xref:../../dev_guide/projects.adoc#dev-guide-projects[interact
 with projects] using xref:../../cli_reference/index.adoc#cli-reference-index[the CLI] or the
 xref:../infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[web console].
+
+ifdef::openshift-online[]
+[[projects-hibenation]]
+=== Project Hibernation
+In {product-title} Starter, projects hibernate after 30 minutes of inactivity.
```

Comment:
@ahardin-rh @yasun1 @damemi 
"In {product-title} Starter, inactive projects are placed in an idled state with resources scaled to zero.  A project is considered inactive when that project's cumulative network traffic received over 30 minutes is below a configured threshold.  Upon receiving any network activity, an idled project's resources are un-idled (scaled back up)."